.drone/drone.jsonnet

```diff
@@ -672,6 +677,54 @@ local manifest_ecr(apps, archs) = pipeline('manifest-ecr') {
       },
     ],
   },
+  pipeline('update-loki-helm-chart-on-loki-release') {
+    local configFileName = 'updater-config.json',
+    depends_on: ['manifest'],
+    image_pull_secrets: [pull_secret.name],
+    trigger: {
+      // wee need to run it only on Loki tags that starts with `v`.
+      ref: ['refs/tags/v*'],
+    },
+    steps: [
+      {
+        name: 'check-version-is-latest',
+        image: 'alpine',
+        when: onTag,
+        commands: [
+          'apk add --no-cache bash git',
+          'git fetch --tags',
+          "latest_version=$(git tag -l 'v[0-9]*.[0-9]*.[0-9]*' | sort -V | tail -n 1 | sed 's/v//g')",
+          'RELEASE_TAG=$(./tools/image-tag)',
+          'if [ "$RELEASE_TAG" != "$latest_version" ]; then echo "Current version $RELEASE_TAG is not the latest version of Loki. The latest version is $latest_version" && exit 78; fi',
+        ],
+      },
+      {
+        name: 'prepare-helm-chart-update-config',
+        image: 'alpine',
+        depends_on: ['check-version-is-latest'],
+        commands: [
+          'RELEASE_TAG=$(./tools/image-tag)',
+          'echo $PLUGIN_CONFIG_TEMPLATE > %s' % configFileName,
```

Comment:
Where's the template defined? We'd also have to increase the Helm chart version.

Reply:
Just saw that it does update the chart version. I'm not sure I understand how this works though.

Reply:
Hey @jeschkies .
It's Grafana's internal tool that does:
1. update values in yaml file
2. bump minor version of helm chart
3. add an entry to changelog.md file
4. re-generate the docs using `helm-docs` tool